user-config-example.yml

```diff
@@ -0,0 +1,6 @@
+test:
+   pulse:
+    username:       !env PULSE_USERNAME
```

Comment:
We generally set the examples to be `...`. See https://github.com/taskcluster/taskcluster-queue/blob/master/user-config-example.yml for example!

Reply:
ah, i will correct it. And maybe also update the `user-config-example.yml` in tc-queue